Source/DataSources/GeometryHeightProperty.js

```diff
@@ -0,0 +1,183 @@
+define([
+    '../Core/ApproximateTerrainHeights',
+    '../Core/defined',
+    '../Core/defineProperties',
+    '../Core/Check',
+    '../Core/Event',
+    '../Core/GeometryOffsetAttribute',
+    '../Scene/HeightReference',
+    './createPropertyDescriptor',
+    './Property'
+], function(
+    ApproximateTerrainHeights,
+    defined,
+    defineProperties,
+    Check,
+    Event,
+    GeometryOffsetAttribute,
+    HeightReference,
+    createPropertyDescriptor,
+    Property) {
+    'use strict';
+
+    /**
+     * A {@link Property} which computes height or extrudedHeight for a {@link CorridorGraphics}, {@link EllipseGraphics}, {@link PolygonGraphics} or {@link RectangleGraphics}.
+     *
+     * @alias GeometryHeightProperty
+     * @constructor
+     *
+     * @param {Property} [height] A Property specifying the height in meters
+     * @param {Property} [heightReference=HeightReference.NONE] A Property specifying what the height is relative to.
+     *
+     * @example
+     * // A 40 meter tall extruded polygon clamped to terrain
+     * var redPolygon = viewer.entities.add({
+     *     polygon : {
+     *         hierarchy : polygonPositions,
+     *         material : Cesium.Color.RED,
+     *         height : Cesium.GeometryHeightProperty(40.0, Cesium.HeightReference.RELATIVE_TO_GROUND),
+     *         extrudedHeight : Cesium.GeometryHeightProperty(0.0, Cesium.HeightReference.CLAMP_TO_GROUND)
+     *     }
+     * });
+     *
+     * // An flat ellipse at 20 meters above terrain (does not contour to terrain)
+     * var blueEllipse = viewer.entities.add({
+     *     position: Cesium.Cartesian3.fromDegrees(-110.0, 43.0),
+     *     ellipse : {
+     *         semiMinorAxis : 300.0,
+     *         semiMajorAxis : 200.0,
+     *         material : Cesium.Color.BLUE,
+     *         height : new Cesium.GeometryHeightProperty(20.0, Cesium.HeightReference.RELATIVE_TO_GROUND)
+     *     }
+     * });
+     */
+    function GeometryHeightProperty(height, heightReference) {
+        this._height = undefined;
+        this._heightSubscription = undefined;
```

Comment:
Unless I'm mistaken, this is unused.

Reply:
Nevermind, brain fart.  Sorry for the noise.  It's been a while since I was in Property code.